ducktape/tests/loader.py

```diff
@@ -141,7 +173,12 @@ def import_modules(self, file_list):
                 module_name = '.'.join(path_pieces)
                 # Try to import the current file as a module
                 try:
-                    module_list.append(importlib.import_module(module_name))
+                    module_list.append(
+                        {
```

Comment:
Given how this is actually used:

```
        modules_and_files = self.import_modules(test_files)
        for mf in modules_and_files:
            test_context_list.extend(self.expand_module(mf))
```

it might be better to just use tuples, which admittedly leave a bit of the info implicit, but can be clearer and less awkward:

```
        for module,file in self.import_modules(test_files):
            test_context_list.extend(self.expand_module(module, file))
```